### PR TITLE
Add typewriter effect for pipinstall on homepage

### PR DESCRIPTION
**Pull request by SSS-Says-Snek**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -44,9 +45,8 @@ jobs:
           pip install pygame-ce --pre
       - name: Clone and move docs
         run: |
-          python utils/generate_docs.py
-
-      # Build website
+        # Build actual site
+        uses: actions/checkout@v3
```
Comment (novialriptide) [inline .github/workflows/build.yml:49]: Unrelated changes. Please remove.

**Review by novialriptide — CHANGES_REQUESTED** [review]
`package-lock.json` has been deleted as well. Please add back.

File: .github/workflows/build.yml (hunk 1 above)
Comment (SSS-Says-Snek) [reply .github/workflows/build.yml:49]: Something happened while fixing merge conflicts, probably cuz of weird girl stuff

Comment (SSS-Says-Snek) [reply .github/workflows/build.yml:49]: Git*

**Comment by SSS-Says-Snek** [comment]
It hasn't been deleted, stuff was just modified (maybe my setup)

**Comment by SSS-Says-Snek** [comment]
Resolved

**Review by novialriptide — APPROVED** [review]
LGTM, thanks!
